modules/layers/src/text-layer/multi-icon-layer/multi-icon-layer-fragment.glsl.js

```diff
@@ -47,12 +48,13 @@ void main(void) {
 
   // Take the global opacity and the alpha from vColor into account for the alpha component
   float a = alpha * vColor.a;
-
+  
+  vec4 color = vec4(vColor.rgb, a);;
   if (a < alphaCutoff) {
-    discard;
+    color = transparentColor;
   }
 
-  gl_FragColor = vec4(vColor.rgb, a);
+  gl_FragColor = color;
```

Comment:
Does this changes the default (no background) behavior in auto highlight?

Reply:
This is fixed in the updated shader.